lib/nexmo_developer/spec/services/translator/translator_coordinator_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Translator::TranslatorCoordinator do
+  it 'accepts an array of docs paths for initialization' do
```

Comment:
can we use a `let` for the paths and `subject` for the coordinator? It's a bit more DRY and in the future, we'll only need to change them in one place